### PR TITLE
Modelregistry: fix bug in ExtractPaths

### PR DESCRIPTION
**Pull request by SeanCondon**
Also removed a duplicate function

Convert enum (from identityref) as numbers in to strings

### CODE REVIEW

**Comment by adibrastegarnia** [comment]
@SeanCondon 
Perhaps you should run all the integration tests  locally (if you haven't done already) a few times and make sure they pass.

**Comment by SeanCondon** [comment]
I will now - just wanted to push it first

File: pkg/modelregistry/jsonvalues/jsonToValues.go [hunk 1]
```diff
@@ -246,16 +243,42 @@ func handleAttribute(value interface{}, parentPath string, modelROpaths modelreg
 			return nil, fmt.Errorf("unable to locate %s in model", parentPath)
 		}
 		modeltype = subPath.Datatype
+		enum = subPath.Enum
 	} else {
 		modeltype = pathElem.ValueType
+		//enum = pathElem.
```
Comment (adibrastegarnia) [inline pkg/modelregistry/jsonvalues/jsonToValues.go:249]: If it is not needed, perhaps you should remove this line.

**Review by adibrastegarnia — APPROVED** [review]
@SeanCondon 
When you are done with testing,

**Review by ray-milkey — APPROVED** [review]

**Pull request merged**